targets/riscv64.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Perhaps you could make a parent riscv.json that is inherited by riscv32.json and riscv64.json?
I agree it's a good idea to split. Using a common parent might avoid some bugs where some things are unintentionally different between RV32 and RV64.

Reply:
done